trajopt_utils/src/clock.cpp

```diff
@@ -29,8 +33,8 @@ double GetClock()
 {
   struct timeval startTimeStruct;
   unsigned long int curTime;
-  gettimeofday(&startTimeStruct, NULL);
-  curTime = startTimeStruct.tv_sec * (long unsigned int)(1e6) + startTimeStruct.tv_usec;
-  return (1e-6) * (curTime - startTime);
+  gettimeofday(&startTimeStruct, nullptr);
+  curTime = static_cast<unsigned long>(startTimeStruct.tv_sec * 1e6l + startTimeStruct.tv_usec);
```

Comment:
Is this supposed to be 1e6**1**?

Reply:
That is interesting GitHub make it look like the number one but it is actually the letter l (Long).

Reply:
Ah. That makes sense. Then I'd say it's good to merge.